src/Telemetry/AzureDevOpsTimeline/BuildLogScraper.cs

```diff
@@ -0,0 +1,59 @@
+using Castle.Core.Logging;
+using Microsoft.DotNet.Internal.AzureDevOps;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
```

Comment:
Looks like we don't need few of specified using

Reply:
removed